IdentityCore/src/telemetry/request_telemetry/MSIDLastRequestTelemetry.m

```diff
@@ -47,7 +47,9 @@ - (instancetype)initWithCoder:(NSCoder *)decoder
         self.apiId = (NSInteger)[decoder decodeFloatForKey:kApiId];
         
         NSString *uuIdString = [decoder decodeObjectForKey:kCorrelationID];
-        self.correlationId = ![NSString msidIsStringNilOrBlank:uuIdString] ? [[NSUUID UUID] initWithUUIDString:uuIdString] : nil;
+        if ([NSString msidIsStringNilOrBlank:uuIdString]) return nil;
```

Comment:
looks like a change of behavior. Before we would just set correlationId to nil if uuIdString is nil. Now we return nil. Is this expected?

Reply:
fixed